version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.4.9-SNAPSHOT"
+version in ThisBuild := "0.4.10-SNAPSHOT"
```

Comment:
Since it is changing schemas, I would rather bump the version, but there is no rule for this package as far as I know. In any case, we are not following semantic versioning!